app/mailers/user_mailer.rb

```diff
@@ -9,23 +9,30 @@ class UserMailer < ActionMailer::Base
   end
 
   # checks the status of the current reservation and sends the appropriate email
-  # receipt forces a check out receipt to be sent
-  def reservation_status_update(reservation, receipt = false) # rubocop:disable all
-    set_app_config
-
-    @reservation = reservation
-    @status = @reservation.human_status
+  # force overrides and sends the specified email
 
-    return if !receipt && @status == 'returned' && @reservation.overdue &&
-              @reservation.equipment_model.late_fee == 0
+  VALID_STATUSES = %w(checked\ out denied due\ today missed overdue
```

Comment:
Ohh... that's probably why Rubocop didn't complain; some of the strings have spaces in them. Sorry, let's revert this and go back to using brackets (my bad!).